rust/worker/src/log/log.rs

```diff
@@ -338,23 +338,25 @@ impl Debug for InternalLogRecord {
 // This is used for testing only
 #[derive(Clone, Debug)]
 pub(crate) struct InMemoryLog {
-    collection_to_log: HashMap<String, Vec<Box<InternalLogRecord>>>,
+    collection_to_log: HashMap<String, Vec<InternalLogRecord>>,
```

Comment:
I don't think this change is propagated to users?

Reply:
It was not.